modules/UI.js

```diff
@@ -0,0 +1,74 @@
+export default class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  static displayBooks() {
+    const books = Book.getBooks();
+    if (Book.checkEmpty()) return;
+    books.forEach((book) => Book.addBookToList(book));
+  }
+
+  static addBookToList(book) {
+    const list = document.querySelector('#book-list');
+
+    const row = document.createElement('tr');
+
+    row.innerHTML = `
+      <td>"${book.title}" by ${book.author}</td>
+      <td><a href="#" class="btn btn-outline-primary delete">remove</a></td>
+      `;
+
+    list.appendChild(row);
+  }
+
+  static deleteBook(el) {
+    if (el.classList.contains('delete')) {
+      el.parentElement.parentElement.remove();
+    }
+  }
+
+  static getBooks() {
+    const books = JSON.parse(localStorage.getItem('books')) || [{
+      title: 'The Great Gatsby', author: 'F. Scott Fitzgerald',
+    }, {
+      title: 'Jane Eyre', author: 'Charlotte Bronte',
+    }];
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Book.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(removed) {
+    const books = Book.getBooks();
+    const m = document.querySelectorAll('tr');
+    let counter = 0;
+    for (let i = counter; i < m.length; i += 1) {
+      if (m[i] === removed) {
+        counter = i;
+        break;
+      }
+    }
+    books.splice(counter, 1);
+    localStorage.setItem('books', JSON.stringify(books));
+    Book.checkEmpty();
+  }
+
+  static checkEmpty() {
+    const books = Book.getBooks();
+    if (books.length === 0) {
+      const isEmptyMessage = document.createElement('div');
+      isEmptyMessage.innerText = 'Add books to your list, there are no books to display';
+      document.querySelector('#book-list').appendChild(isEmptyMessage);
+      return true;
+    }
```

Comment:
- [ ] As required by the previous code reviewer I recommend to use the arrow methods syntax instead of class methods so you can make it like this and so on for all the methods here! 🙏🏻🔥😊💯✨it is very important to follow the requirements because it is evaluating your understanding to the topic 😉💯💯🔥🙏🏻🙏🏻

![image](https://user-images.githubusercontent.com/58553711/215290394-8287541d-6793-4de4-8499-cba39c715aa4.png)



```
static const checkEmpty = () => {
body....
}
```